text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,196 @@
+---
+rfc pr: [#347](https://github.com/aws/aws-cdk-rfcs/pull/347)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/66
+---
+
+# CDK Self Managed Stack Set Support
+
+As a CDK user, I would like to define stack set infrastructure through native CDK constructs,
+ and create/update self-managed stack sets through CDK deploy command.
+
+* **Original Author(s):** @linsona
+* **Tracking Issue**: #66
+* **API Bar Raiser**: @skinny85
+
+## Working Backwards
+
+### CHANGELOG
+
+* feat(assembly): add stack set artifact type and schema
+* feat(core): add stack set core construct
+* feat(cli): add logic for cli commands such as `cdk deploy` to collect stack set artifact types
+* feat(bootstrap): add default stack set roles for CDK deployments
+
+### README
+
+The `cdk.StackSet` construct in CDK defines AWS resources in a stack set and stack set
+ configuration/deployment preferences.
+ This is different from the `cdk.CfnStackSet` construct, which requires a CloudFormation stack that then deploys a CloudFormation stack set.
+ The `cdk.StackSet` construct creates the stack set directly if it does not exist,
+or updates a stack set and existing stack set instances while monitoring the stack set operation.
+
+#### Prerequisites
```

Comment:
Let's divide this section a little more finely. Let's say that there are a few ways you can prepare your parent and child accounts ford deploying StackSets.

1. You can use the `cdk bootstrap` command. Show the exact invocations you need in both the parent and child accounts. And show how it looks in the code.
    - Describe how you can pass the `--qualifier` option to `cdk bootstrap`, and how is that reflected in the code (perhaps there's a `qualifier` property when creating a `cdk.StackSet`?).
2. You can create the Roles manually. Describe what the Roles have to be (names, trust policies permission policies). Then, show how is that reflected in the code (probably properties like `administratorRoleName` and `executionRoleName` of `cdk.StackSet`).

Reply:
Broke up the section into those two methods